taskmanager/features/report.feature

```diff
@@ -0,0 +1,29 @@
+# language: en
+Feature: Generate a report
+  In order to export tasks with details
+  As a user or system
+  I want to generate various of reports
+
+  Scenario: Generate report for TODO tasks
+    Given There is no reports
+    And there are "2" tasks with "TODO" status
+    When I generate report with criteria "unfinished"
```

Comment:
Maybe would better to define criteria as a list.
```gherkin
When I generate report with the following criteria:
  | Key        | Value          |
  | status    | unfinished  |
```
etc. thanks to that you would be able  to define more than one condition